.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Tests / Code Coverage
+#  Tests / Code Coverage workflow runs unit tests and uploads a code coverage report
+#  This workflow is run on pushes to main & every Pull Requests where a .go, .mod, .sum have been changed
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@master
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main'"
+
+  test-coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - name: display go version
+        run: go version
+      - name: test & coverage report creation
+        run: make test-unit-cover
+      - name: filter out DONTCOVER
+        run: |
```

Comment:
동삼님. 실제 tests를 돌리는 커맨드는 어디에 있나요?
이 workflow는 Testing과 Code Coverage 작업을 해줄 것으로 보이는데, tests를 실행하는 구문은 없는 것 같습니다.

Reply:
Test and coverage measurements are made through `make test-unit-cover`, 
(Since github is globally public, I think it would be better to communicate in English unless it is a special situation)